index.js

```diff
@@ -10,34 +10,28 @@ export const ContextUser = createContext(true);
 
 const root = createRoot(document.getElementById('app'));
 
-class App extends Component {
-  state = {
-    isAuthenticated: false
-  }
+const App = () => {
+  const [isAuthenticated, setIsAuthenticated] = useState(false)
 
-  async componentDidMount() {
+  useEffect(async () => {
```

Comment:
it is forbidden to pass async functions to `useEffect` (see warning in console and make as advised)
![image](https://user-images.githubusercontent.com/98347942/207091306-d4e96de3-5968-4cf5-b478-928d4e29e8b2.png)

Reply:
Thanks, I saw. Fixed it.